src/polyswarmd/websockets/messages.py

```diff
@@ -0,0 +1,178 @@
+import json
```

Comment:
```suggestion
try:
    import ujson as json
except ImportError:
    import json
```
Switching to ujson since it can serialize data a bit faster